sys/include/xtimer.h

```diff
@@ -73,8 +73,10 @@ typedef void (*xtimer_callback_t)(void*);
  */
 typedef struct xtimer {
     struct xtimer *next;         /**< reference to next timer in timer lists */
-    uint32_t target;             /**< lower 32bit absolute target time */
-    uint32_t long_target;        /**< upper 32bit absolute target time */
+    uint32_t offset;             /**< lower 32bit offset time */
+    uint32_t long_offset;        /**< upper 32bit offset time */
+    uint32_t start_time;         /**< lower 32bit absolute start time */
+    uint32_t long_start_time;    /**< upper 32bit absolute start time */
```

Comment:
Hah! I initially started ztimer to reduce the size of xtimer_t. :)

Reply:
Yes this is not optimal, but I think we can live with it for now. 
Actually it could be possible to just have a reference to the long_start_time, always assume a 0 start_time and adapt the offsets accordingly... but that will create other problems with the extension again^^
For comparison of xtimer/ztimer benches this should be good for ztimer though ;)

Reply:
> For comparison of xtimer/ztimer benches this should be good for ztimer though ;)

yup. :smile: `sizeof(xtimer)` is 28 with this PR vs. 16 in ztimer.

Reply:
@MichelRottleuthner good suggestion. Might be able to consider as a next PR :) Adding long_start_time is kinda quick, after-thought to merge this PR for this release.

Reply:
Good, I'm totally fine with leaving that for a future PR